gyucheol/5일차/이상한_문자_더하기.js

```diff
@@ -0,0 +1,18 @@
+function solution(s) {   
+    let arr = s.split(' ')
+        let result = [];
+        console.log(arr)
+        for(let i = 0; i<arr.length; i++){
+            let words = arr[i].split('').map((word,idx)=>{ 
+            if(idx % 2 === 0){
+                return word.toUpperCase()
+            } else return word.toLowerCase();
+        }
+    ).join(''); console.log(words);
+    
+            result.push(words)
+    }
+    
+    return result.join(' ')
```

Comment:
console.log는 제출하는 코드에서 삭제해줘야 합니다 ㅎㅎ
모든 배열이나 객체는 const 이용해서 풀어주세요~
```suggestion
    const arr = s.split(' ');
    const result = [];

    for (let i = 0; i<arr.length; i++) {
        const words = arr[i].split('').map((word,idx) => { 
            if (idx % 2 === 0) {
                return word.toUpperCase();
            } else {
                return word.toLowerCase();
            }
        }).join('');
    
        result.push(words);
    }
    
    return result.join(' ')
```